neurom/features/_impl.py

```diff
@@ -178,3 +181,55 @@ def principal_directions_extents(neurons, neurite_type=TreeType.all, direction='
     n = 0 if direction == 'first' else (1 if direction == 'second' else 2)
     return (_mt.principal_direction_extent(t)[n]
             for t in iter_neurites(neurons, filt=_ttc(neurite_type)))
+
+
+def as_neuron_list(func):
```

Comment:
This also seems to duplicate existing behaviour. I will see if it can be removed in a later, separate pull request.